code/network/multi_obj.cpp

```diff
@@ -703,19 +731,28 @@ void multi_ship_record_do_rollback()
 
 	nprintf(("Network","At least one multiplayer rollback shot is being simulated this frame.\n"));
 
+	bool continue_rollback = true;
+
 	do {
 		// move all ships to their recorded positions
 		multi_oo_restore_frame(frame_idx);
 
 		// push weapons forward for the frame (weapons do not get pushed forward for the first frame of their existence)
-		multi_oo_simulate_rollback_shots(frame_idx);
+		continue_rollback = (multi_oo_simulate_rollback_shots(frame_idx));
```

Comment:
A bit of a nitpick, but the extra parenthesis should be removed. clang tends to complain about this too so I'm a little surprised that clang-tidy let it through.

Reply:
My reasoning was always to help indicate that it is a bool assignment.  I'll refrain from now on.